pkg/regexp/regexp_dontprecompile.go

```diff
@@ -0,0 +1,6 @@
+//go:build !regexp_precompile
+// +build !regexp_precompile
+
+package regexp
+
+var precompile = false
```

Comment:
should this be const?

Reply:
Yes (in both cases); this decision should have zero run-time cost.